### ISSUE TITLE
Support inline templates with hbs helper

### ISSUE THREAD

**Issue by jelhan** [ember-codemods/ember-no-implicit-this-codemod#262]
Implicit `this` fallback is deprecated in Ember canary: https://deprecations.emberjs.com/v3.x/#toc_this-property-fallback

While trying to address the deprecation in Ember Bootstrap I noticed that many tests were still relying on implicit `this` fallback. The codemod does not support templates written with `hbs` helper yet. So basically not support for tests yet. But for libraries like Ember Bootstrap with a huge test suite updating them by hand means a lot of work. Even after investing two hours to fix some of them, I still face 119 lint errors. Please find more details here: https://github.com/kaliber5/ember-bootstrap/pull/1421/files

Long story short: To address the deprecation in addons it would be very helpful if the codemod could support inline templates used in tests as well.
